app/model/command/ImportCampaignFromCAPICommand.scala

```diff
@@ -29,7 +29,10 @@ trait CAPIImportCommand extends Command {
   }
 
   def deriveContentType(apiContent: ApiContent) = {
-    apiContent.tags.find(_.`type` == TagType.Type).map(_.webTitle).getOrElse(UnableToDetermineContentType)
+    apiContent.tags
+      .find(_.`type` == TagType.Type)
+      .map(_.webTitle)
+      .getOrElse(throw new RuntimeException(s"Unable to derive content type from ${apiContent.id}"))
```

Comment:
Why are you throwing a runtime exception here?

Reply:
Because I haven't removed all the exceptions yet. Actually, this was meant to replace the removal of `CommandError` but I put `CommandError` back in as there were too many usages, but didn't revert this!